packages/block-library/src/latest-posts/edit.js

```diff
@@ -328,131 +327,239 @@ function Controls( { attributes, setAttributes, postCount } ) {
 					/>
 				</ToolsPanelItem>
 			</ToolsPanel>
-			<PanelBody title={ __( 'Featured image' ) }>
-				<ToggleControl
-					__nextHasNoMarginBottom
+			<ToolsPanel
+				label={ __( 'Featured image' ) }
+				resetAll={ () =>
+					setAttributes( {
+						displayFeaturedImage: false,
+						featuredImageAlign: undefined,
+						featuredImageSizeSlug: undefined,
+						featuredImageSizeWidth: undefined,
+						featuredImageSizeHeight: undefined,
+						addLinkToFeaturedImage: false,
+					} )
+				}
+				dropdownMenuProps={ dropdownMenuProps }
+			>
+				<ToolsPanelItem
+					hasValue={ () => displayFeaturedImage !== false }
```

Comment:
```suggestion
					hasValue={ () => !! displayFeaturedImage }
```

This would be a simpler way to write it.

Reply:
Changes have been pushed.